src/status_im/utils/gfycat/core.cljs

```diff
@@ -12,3 +13,14 @@
     (native-module/generate-gfycat public-key)))
 
 (def generate-gfy (memoize build-gfy))
+
+(re-frame/reg-fx
+ :insert-gfycat-fx
+ (fn [[public-key path]]
+   (native-module/generate-gfycat-async public-key #(re-frame/dispatch [:set-generated-gfycat path %]))))
```

Comment:
call the event something like "gfycat-generated" or "generate-gfycat-callback", something that happened not an action name.